### ISSUE TITLE
Fix uploaded forms view feature

### ISSUE THREAD

**Issue by Jillzey** [OpenCLTBrigade/cmpd-holiday-gift#42]
After a form is uploaded there needs to be a way to view that form.

**Comment by Jillzey** [comment]
When I tried to test this. None of them opened. I got a message saying that it "Failed to load pdf document". I test about 10 random one and got the message every time. 

Here is a screenshot
![screen shot 2016-11-21 at 9 07 51 am](https://cloud.githubusercontent.com/assets/7969425/20485583/47178a6e-afca-11e6-86d6-2c15c1accb64.png)

**Comment by AtnNn** [comment]
I get that same error using Chrome. A work-around is to Save As and give it a `pdf` extension.

Viewing PDF attachments works for me in Firefox.

I believe this could be fixed properly by sending a different `Content-Type` or `Content-Disposition` header.

**Comment by Jillzey** [comment]
I don't get an option to Save As. But the function does work in Safari. 

Thanks!

**Comment by AndrewNatoli** [comment]
Should we consider this resolved? @AtnNn @Jillzey

**Comment by AtnNn** [comment]
I've been working a fix (see bdcbd3e) but I moved onto other things because we discussed last week that getting it working in Chrome is not a priority.

**Comment by AndrewNatoli** [comment]
I can dig it. Thanks for the update :D


On November 29, 2016 at 7:42:54 PM, Laurin Etienne (notifications@github.com)
wrote:

I've been working a fix (see bdcbd3e
<https://github.com/CodeForCharlotte/cmpd-holiday-gift-backend/commit/bdcbd3eff21035671c084a4384b6ae20187a6594>)
but I moved onto other things because we discussed last week that getting
it working in Chrome is not a priority.

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub
<https://github.com/CodeForCharlotte/cmpd-holiday-gift-backend/issues/42#issuecomment-263747712>,
or mute the thread
